api/python/provisioner/commands/__init__.py

```diff
@@ -431,6 +438,34 @@ def run(self, *args, targets=ALL_MINIONS, dry_run=False, **kwargs):
 
         return self._run(params, targets)
 
+@attr.s(auto_attribs=True)
+class PillarDump(CommandParserFillerMixin):
+    input_type: Type[inputs.PillarKeysList] = inputs.PillarKeysList
+    _run_args_type = RunArgsPillarDump
+
+    @classmethod
+    def from_spec(
+        cls, input_type: str = 'PillarKeysList'
+    ):
+        return cls(input_type=getattr(inputs, input_type))
+
+
+    def run(
+        self, *args, targets: str = ALL_MINIONS, local: bool = False,
+        **kwargs
+    ):
+        res = PillarGet.run(self)
+        os.makedirs(f"{PRVSNR_CORTX_CONFIG_DIR}", exist_ok=True)
+
+        resp_loaded = json.loads(json.dumps(res).replace('null', '""'))
+        for key in resp_loaded.keys():
+            resp_loaded[key] = ({k:v
+                 for k, v in resp_loaded[key].items() if not k == "mine_functions"})
+        resp_dumped = json.dumps(resp_loaded, indent=4, sort_keys=True)
+        resp_unload = open(f'{PRVSNR_CORTX_CONFIG_FILE}', "w").write(resp_dumped)
```

Comment:
that should be closed as well, please use pathlib (PRVSNR_CORTX_CONFIG_FILE is already a Path object) instead, e.g.:
```
PRVSNR_CORTX_CONFIG_FILE.write_text(resp_dumped)
```

Reply:
Addressed. Thanks